pyproject.toml

```diff
@@ -54,3 +54,23 @@ changelog = "https://github.com/jazzband/django-constance/releases/"
 
 [tool.setuptools.packages.find]
 include = ["constance*"]
+
+
+[tool.ruff]
+# Same as Black.
+line-length = 88
+indent-width = 4
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
+skip-magic-trailing-comma = false
+line-ending = "auto"
+
+[tool.ruff.lint]
+select = [
+    "I", # isort
+]
+
+[tool.ruff.lint.isort]
+force-single-line = true
```

Comment:
What is the reason to do so?

For me
**from datetime import date, datetime, time, timedelta**
is better then 
**from datetime import date
from datetime import datetime
from datetime import time
from datetime import timedelta**

Reply:
It's the trend nowadays because it's easy to remove a line when the import is removed.

Reply:
The single-line format reduces merge conflicts significantly

Reply:
I feel the reason, thank you. I like this trend.